.github/workflows/weekly.yml

```diff
@@ -1,6 +1,7 @@
 name: Weekly package builds
 
 on:
+  pull_request:
```

Comment:
isn't it enough to trigger it when it's merged into main?

Reply:
I just wanted to catch this before this is merged :)
I'll revert it now.